src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The bulldozer stopped working");

```

Comment:
Similarly, the stop message should be more informative. Change it to 'Bulldozer stopped working'.